### PR TITLE
Tweaks and fixes

### PR DESCRIPTION
**Pull request by maudnals**
As discussed with @johnidel and @csharrison.

### CODE REVIEW

**Comment by csharrison** [comment]
cc @johnivdel , johnidel is the wrong username :) I'll try to review this today

File: README.md [hunk 1]
```diff
@@ -148,15 +148,16 @@ Without a Feature Policy, a top-level document and cooperating iframe could recr
 Conversion Registration
 -----------------------
 
-This API will use a similar mechanism for conversion registration as the
+This API will use a similar mechanism for conversion registration similar to the
```
Comment (johnivdel) [inline README.md:151]: nit: maybe drop the first similar in this line

File: README.md [hunk 2]
```diff
@@ -204,15 +206,15 @@ from 0-5 (~2.6 bits of information)
...
+When the browser receives a conversion registration on a URL matching
+the `conversiondestination` eTLD+1, it looks up all impressions in storage that
+match <`reportingorigin`, `conversiondestination`>.
 
 The most recent matching impression is given an `attribution-credit` of value 100. All other matching impressions are given an `attribution-credit` of value of 0.
 
 For each matching impression, schedule a report. To schedule a report,
 the browser will store the 
- {reporting origin, conversiondestination domain, impression data, [decoded](#metadata-encoding) conversion-metadata, attribution-credit} for the impression.
+ {reporting origin, `conversiondestination` domain, impression data, [decoded](#metadata-encoding) conversion-metadata, attribution-credit} for the impression.
```
Comment (johnivdel) [inline README.md:217]: Should we use `impression-data` as it is also an attribute?

**Comment by johnivdel** [comment]
lgtm as well % a couple comments

File: README.md (hunk 2 above)
Comment (maudnals) [reply README.md]: Yes, fixed - also edited `reportingorigin`.

File: README.md (hunk 1 above)
Comment (maudnals) [reply README.md]: Right, fixed.

**Comment by maudnals** [comment]
@johnivdel Ready for review - thanks!

**Comment by csharrison** [comment]
Note: Travis is complaining about bad builds due to this known bug:
https://github.com/tabatkins/bikeshed/issues/1746#issuecomment-671514945

**Comment by tabatkins** [comment]
Fixed, sorry about that.

**Pull request merged**